src/libraries/component-time-scroll-view/YAxisTicks.ts

```diff
@@ -69,12 +71,12 @@ const makeStep = (raw: number, base: number, scale: number): Step => {
     const trimmed = raw - (TRUNCATE_UNCHANGED_HIGHER_ORDER_DIGITS ? base : 0)
     const scaled = Math.trunc(trimmed * Math.pow(10, -scale))
     const isMajor = scaled % 10 === 0
-    const label =  Math.abs(scale) > 3 ? `${(scaled/10).toFixed(1)}e${scale + 1}` : `${scaled * Math.pow(10, scale)}`
+    const label =  Math.abs(scale) > 3 ? `${(scaled/10).toFixed(1)}e${scale + 1}` : `${Math.round(scaled * Math.pow(10, scale) * 1e9) / 1e9}` // use precision rounding
```

Comment:
I would've hoped the truncation above would take care of this... oh well, guess not :(

Seems a good fix!